Source/mainmenu.cpp

```diff
@@ -24,6 +24,9 @@ void mainmenu_refresh_music()
 
 void __stdcall mainmenu_change_name(int arg1, int arg2, int arg3, int arg4, char *name_1, char *name_2)
 {
+#ifdef SWITCH
+	strcpy(name_2, "Switcher");

```

Comment:
This will be solved in a better way once https://github.com/diasurgical/devilutionX/pull/419 is merged.

Reply:
This is here because the touch keyboard on Switch can be cancelled or fail with no keyboard input coming from it. In that case, the keyboard cannot be brought up again, and the game cannot be started unless there's some hero name.

Reply:
Yes, but #419 adds some better default names :)